src/App.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(
+    (movie) => movie.title.toLowerCase().includes(query.toLowerCase())
+      || movie.description.toLowerCase().includes(query.toLowerCase()),
+  );
```

Comment:
maybe it would be better to put lowercase conversion in a separate variable

Reply:
and also it would be better to trim query to get better search results. the mentor left me a comment about this